### PR TITLE
improvements for Live-TV

### PR DESCRIPTION
**Pull request by walterheisenberg**
- fixed recording Icon in Compact and Standard Bar
- make recordingIcon consistent over all windows
- changed remaining time in Live-TV from "00:31" to "+31 min". This is IMHO better to read, because below is the endtime of the next show  e.g. "00:45" (ends at 00:45 am)
- added info in Fullscreen, if another channel is recorded
- and some more 

@mistervee 
do you mind having a look?
Thx

### CODE REVIEW

**Comment by walterheisenberg** [comment]
Difference of my time remaining var:
If a Show on a channel has e.g. 1 hour and 38 minutes left, then this is the output of the different vars:
**PlayerTimeRemainingVar:**
1 hour
38 minutes left
**PlayerTimeRemainingSmallVar:**
1 hour 38 minutes left
**PlayerTimeRemainingTVVar:**
1h 38min

I preferred the last one for Live TV, it looks better because below this string is the endtime of the next show, which can sometimes be confusing (see my first post)
http://imgur.com/a/RInSm

In the picture, you see the recording info, if you are recording another channel. The recording Icon is also working again.
Will edit my comments in the code and push it to github.

File: 16x9/MyPVRTimers.xml [hunk 1]
```diff
@@ -196,7 +202,13 @@
 							<selectedcolor>$VAR[ThemeLabelColor]</selectedcolor>
 							<label></label>
 							<visible>ListItem.IsRecording</visible>
-						</control>
+						</control>-->
+                        <control type="group">
+                            <top>39</top>
+                            <left>69</left>
+                            <!--<left>479</left>-->
```
Comment (mikesilvo164) [inline 16x9/MyPVRTimers.xml]: Needs to be deleted

File: 16x9/variables.xml [hunk 2]
```diff
@@ -301,7 +301,7 @@
 	</variable>
 	<variable name="VideoPlayerPlotVar">
 		<value condition="!String.IsEmpty(VideoPlayer.PlotOutline)">$INFO[VideoPlayer.PlotOutline]</value>
-		<value>$INFO[VideoPlayer.Plot]</value>
+		<value condition="String.IsEmpty(VideoPlayer.PlotOutline)">$INFO[VideoPlayer.Plot]</value>
```
Comment (mikesilvo164) [inline 16x9/variables.xml]: Fallback does not require a condition

File: 16x9/MyPVRTimers.xml [hunk 3]
```diff
@@ -185,7 +191,7 @@
 							<label></label>
 							<visible>ListItem.HasTimer</visible>
 						</control>
-						<control type="label">
+						<!--<control type="label">
```
Comment (mikesilvo164) [inline 16x9/MyPVRTimers.xml]: If commenting out can it just be deleted?

File: 16x9/MyPVRTimers.xml [hunk 4]
```diff
@@ -118,7 +118,13 @@
 							<selectedcolor>$VAR[ThemeLabelColor]</selectedcolor>
 							<label></label>
 							<visible>ListItem.IsRecording</visible>
-						</control>
+						</control>-->
+                        <control type="group">
+                            <top>39</top>
+                            <left>69</left>
+                            <!--<left>479</left>-->
```
Comment (mikesilvo164) [inline 16x9/MyPVRTimers.xml]: Needs to be deleted

File: 16x9/MyPVRTimers.xml [hunk 5]
```diff
@@ -107,7 +107,7 @@
 							<label></label>
 							<visible>ListItem.HasTimer</visible>
 						</control>
-						<control type="label">
+						<!--<control type="label">
```
Comment (mikesilvo164) [inline 16x9/MyPVRTimers.xml]: If commenting out can it just be deleted?

**Comment by mikesilvo164** [comment]
@walterheisenberg I marked a few things. Mind taking a look?
Is there anything you think I should look at before merging?

File: 16x9/variables.xml (hunk 2 above)
Comment (walterheisenberg) [inline 16x9/variables.xml]: Haven't touched this line ;)

**Comment by walterheisenberg** [comment]
Did the code cleanup you mentioned, without the "VideoPlayerPlot"-thing. I didn't touch this line. Feel free to change it. I'm not (yet) familiar with fallback conditions.

For me, there are no issues left.

**Comment by mikesilvo164** [comment]
Thanks.

**Pull request merged**